python/tests/test_lowlevel.py

```diff
@@ -164,6 +164,36 @@ class TestTableCollection(LowLevelTestCase):
     Tests for the low-level TableCollection class
     """
 
+    def test_skip_tables(self, tmp_path):
+        def tables_are_skipped(tc, tc_skip):
+            return (
```

Comment:
It would be a little better to structure this like
```def assert_tables_are_skipped(tc, tc_skip):
          assert not tc.equals(tc_skip)
          assert tc.equals(tc_skip, ignore_tables=True) 
           ... etc
```
The reasons is that if the tests every fail, the specific problem will be immediately apparent. This way, you'd have to put in some prints to try and figure out which one of the assertions was failing.